data/scripts.js

```diff
@@ -1174,7 +1174,6 @@ let BattleScripts = {
 			if (pokemon.getItem().zMove) return;
 			if (pokemon.template.isMega) return;
 			if (pokemon.template.isPrimal) return;
-			if (pokemon.transformed) return;
```

Comment:
Transformed pokemon can only Dynamax, it can't Gigantamax, in a result of it not being able to Gigantamax, it cannot use the G-Max moves, just the regular Max Moves